src/Component/Templating/src/View/View.js

```diff
@@ -0,0 +1,236 @@
+const RequestAwareInterface = Jymfony.Component.Templating.Helper.RequestAwareInterface;
+const ResponseAwareInterface = Jymfony.Component.Templating.Helper.ResponseAwareInterface;
+const SlotsHelper = Jymfony.Component.Templating.Helper.SlotsHelper;
+
+const promisify = require('util').promisify;
+const streamOut = promisify(require('stream').Writable.prototype.write);
+
+/**
+ * @memberOf Jymfony.Component.Templating.View
+ */
+class View {
+    /**
+     * Constructor.
+     *
+     * @param {module:stream.Writable} output
+     * @param {string|Jymfony.Component.Templating.TemplateReferenceInterface} name
+     * @param {Jymfony.Component.Templating.Engine.JsEngine} engine
+     * @param {Object.<string, *>} parameters
+     */
+    __construct(output, name, engine, parameters = {}) {
```

Comment:
`name` -> `reference`?

Reply:
In the more common usage, the logical name of the template is expected to be used, so `name` is correct. 